app/services/draw_lottery_starter.rb

```diff
@@ -54,11 +55,16 @@ def at_least_one_group
     errors.add(:draw, 'must have at least one group')
   end
 
-  def no_ungrouped_students
-    return unless draw.present? && draw.ungrouped_students?
+  def all_students_grouped
+    return if draw.nil? || draw.all_students_grouped?
```

Comment:
Turns out some of the specs were broken but still passing, either way fixed now.